### PR TITLE
Fix result view for long text answers

### LINKED ISSUE #904: Bug occurs while viewing long text survey results
When viewing survey results, long text response option answers lose their formatting including line breaks.

Nextcloud Forms Version 2.2.4

### PR DESCRIPTION
**Pull request by Chartman123**
Fixes #904

- [x] Summary view
- [x] Individual submissions

Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

File: src/components/Results/Answer.vue [hunk 1]
```diff
@@ -25,7 +25,9 @@
 		<h4 class="question-text">
 			{{ questionText }}
 		</h4>
-		<p>{{ answerText }}</p>
+		<!-- Do not wrap the following line between tags! `white-space:pre-line` respects `\n` but would produce additional empty first line -->
+		<!-- eslint-disable-next-line -->
+		<p class="answer-text">{{ answerText }}</p>
```
Comment (jotoeri) [inline src/components/Results/Answer.vue:30]:
```suggestion
		<p class="answer__text">{{ answerText }}</p>
```

File: src/components/Results/Answer.vue [hunk 2]
```diff
@@ -54,6 +56,10 @@ export default {
 	.question-text {
 		font-weight: bold;
 	}
+
+	.answer-text {
```
Comment (jotoeri) [inline src/components/Results/Answer.vue:60]:
```suggestion
	&__text {
```

**Review by jotoeri — COMMENTED** [review]
Looks good in general. 👍 

Just one thing: For (CSS-)Naming we typically try to use the BEM-Model: http://getbem.com/naming/
So the class-name for the answer-text should better be `answer__text` (see the link for explanation 😉).

And on scss you can use the nested syntax, where the following will resolve as the desired class `answer__text`.
```
.answer {
    &__text {
        some: css;
    }
}
```

-> If you're motivated you can also rename the question-text to a `answer__question-text` to be consistent. There i also did it wrong.... 🙈

**Comment by Chartman123** [comment]
> rename the question-text to a `answer__question-text`

That's where I took my naming from 😆  I've just fixed it and updated my commit

**Comment by jotoeri** [comment]
So - i just tested it and it messes up the countable questions a bit... Probably needs to apply the white-space only on `&__text` class.
grafik

**Comment by Chartman123** [comment]
Ok, I will fix this later :) I tried to spot where the other part is used but didn't find it 🙈

**Review by jotoeri — APPROVED** [review]
Didn't test it again, just hope you did? 😉 🙈

**Comment by Chartman123** [comment]
I checked it for long and short text and radio buttons. 👍🏻

**Comment by jotoeri** [comment]
@Chartman123, wanna rebase here onto current master? And then i'd say you can also merge this. 👍

**Pull request merged**